aggregate/decennial_census/decennial_census_001020.py

```diff
@@ -3,7 +3,7 @@
 from utils.PUMA_helpers import clean_PUMAs
 from internal_review.set_internal_review_file import set_internal_review_files
 
-year_map = {2000: "00", 2010: "10", 2020: "20"}
+year_map = {"2000": "00", "0812": "10", "1519": "20"}
```

Comment:
cool - thanks for updating this, keep the year string consistent

Reply:
having the parameter immediately passed to the mapper makes this sort of update really easy. Having clearly defined mappers as python dictionaries is consistently a good pattern for us in this project